src/plugins/data/public/ui/query_string_input/query_bar_top_row.tsx

```diff
@@ -32,33 +32,34 @@ const QueryStringInput = withKibana(QueryStringInputUI);
 
 // @internal
 export interface QueryBarTopRowProps {
-  query?: Query;
-  onSubmit: (payload: { dateRange: TimeRange; query?: Query }) => void;
-  onChange: (payload: { dateRange: TimeRange; query?: Query }) => void;
-  onRefresh?: (payload: { dateRange: TimeRange }) => void;
+  customSubmitButton?: any;
   dataTestSubj?: string;
+  dateRangeFrom?: string;
+  dateRangeTo?: string;
   disableAutoFocus?: boolean;
-  screenTitle?: string;
+  iconType?: EuiIconProps['type'];
   indexPatterns?: Array<IIndexPattern | string>;
+  indicateNoData?: boolean;
+  isClearable?: boolean;
+  isDirty: boolean;
+  isFill: boolean;
   isLoading?: boolean;
+  isRefreshPaused?: boolean;
+  nonKqlMode?: 'lucene' | 'text';
+  nonKqlModeHelpText?: string;
+  onChange: (payload: { dateRange: TimeRange; query?: Query }) => void;
+  onRefresh?: (payload: { dateRange: TimeRange }) => void;
+  onRefreshChange?: (options: { isPaused: boolean; refreshInterval: number }) => void;
+  onSubmit: (payload: { dateRange: TimeRange; query?: Query }) => void;
+  placeholder?: string;
   prepend?: React.ComponentProps<typeof EuiFieldText>['prepend'];
+  query?: Query;
+  refreshInterval?: number;
+  screenTitle?: string;
   showQueryInput?: boolean;
   showDatePicker?: boolean;
-  dateRangeFrom?: string;
-  dateRangeTo?: string;
-  isRefreshPaused?: boolean;
-  refreshInterval?: number;
   showAutoRefreshOnly?: boolean;
-  onRefreshChange?: (options: { isPaused: boolean; refreshInterval: number }) => void;
-  customSubmitButton?: any;
-  isDirty: boolean;
   timeHistory?: TimeHistoryContract;
-  indicateNoData?: boolean;
-  iconType?: EuiIconProps['type'];
-  placeholder?: string;
-  isClearable?: boolean;
-  nonKqlMode?: 'lucene' | 'text';
-  nonKqlModeHelpText?: string;
   timeRangeForSuggestionsOverride?: boolean;
```

Comment:
While adding a `isFill` prop, I also sorted this long list alphabetically so it's easier to maintain and change in the future.